src/api/app/controllers/application_controller.rb

```diff
@@ -213,7 +213,7 @@ def pass_to_backend(path = nil)
     render_error status: 400, errorcode: 'invalid_xml', message: 'Invalid XML'
   end
 
-  rescue_from APIException do |exception|
+  rescue_from APIError do |exception|
```

Comment:
Please add an explanation why we change inherited exception class to the commit message.

Reply:
that's what I don't like from this review system from github. We "decided" to rename it here, in a discussion between @dmarcoux @coolo @ChrisBr and me. But some how it gets collapsed in a way that I cannot find or link it (at least easily) :/

Reply:
https://github.com/openSUSE/open-build-service/pull/5531#discussion_r208499449 - I was not involved! :)

Reply:
hahaha you are omnipresent @coolo :)